README.md

```diff
@@ -27,13 +27,15 @@ They are converted to a single RML document.
 Note that the keys in `prefixes`, `sources`, and `mappings` have to be unique across all files.
 `base` can only be set once.
 An example can be found in [`test/multiple-input-files`](test/multiple-input-files).
+You can overwrite external references via the `-e`.
+For example, `-e name=John` will replace all occurences of `$(_name)` with `John`.
+Repeat `-e` for multiple references.
```

Comment:
I would add a huge disclaimer that if your reference starts with `_` you should add a `\` before it. This is breaking behavior. Also we need to (separately) make sure the spec is updated accordingly.

Reply:
See my comment at `lib/abstract-generator.js` to make this a non-breaking feature. I would document as such here then as well